### ISSUE TITLE
False error on `<script>` elements inside markup ("duplicate <script> block" / "malformed opening tag")

### ISSUE THREAD

**Issue by MathiasWP** [harshmandan/svelte-check-native#36]
## Summary

Svelte allows `<script>` *elements* in component markup (commonly used for conditionally injecting external scripts, e.g. a CDN loader gated behind `{#if}`). `svelte-check-native` misparses these as component script blocks and reports a false **error**, while `svelte-check` (and the Svelte compiler) accept them fine. A false error makes the tool report a failing check on a codebase that is actually clean.

## Environment

- svelte-check-native **1.0.2** (darwin-arm64), driven via `TSGO_BIN` → `@typescript/native-preview` 7.0.0-dev.20260707.2
- Compared against `svelte-check` **4.7.3** with `svelte` 5.56.5 / TypeScript 6.0.3
- Found while evaluating on a ~2,100-file SvelteKit app

## Repro 1 — plain attributes → "duplicate `<script>` block"

```svelte
<script lang="ts">
	let load = true;

	function handleLoad() {
		load = false;
	}
</script>

{#if load}
	<script defer src="https://example.com/some-script.js" onload={handleLoad}></script>
{/if}
```

```
ERROR "src/ScriptInTemplate.svelte" 10:2 "duplicate <script> block"
```

## Repro 2 — expression attribute → "malformed opening tag"

Same component, but with an expression-valued attribute on the script element:

```svelte
{#if load}
	<script
		defer
		nonce="%sveltekit.nonce%"
		src={`https://example.com/lib@${'1.0.0'}/script.js`}
		onload={handleLoad}
	></script>
{/if}
```

```
ERROR "src/ScriptInTemplateNonce.svelte" 10:8 "malformed opening tag"
```

## Expected

No error — `svelte-check` 4.7.3 reports `0 ERRORS 0 WARNINGS` on both files.

## Aside

Everything else in our evaluation was very impressive — on the full app the tool found every warning the real svelte-check found (perfect recall), at 4.1s warm vs 35.3s for `svelte-check --incremental`. This parse issue (plus #37) is what keeps us from gating CI on it. Happy to test fixes.

**Comment by harshmandan** [comment]
Fixed in release [v1.1.0](https://github.com/harshmandan/svelte-check-native/releases/tag/v1.1.0)
